tests/benchmarks/mbedtls/prj.conf

```diff
@@ -12,15 +12,8 @@ CONFIG_MBEDTLS_DEBUG_LEVEL=4
 
 # TLS configuration
 CONFIG_MBEDTLS=y
-CONFIG_MBEDTLS_BUILTIN=y
```

Comment:
That should be ok, the sample could use the default configuration used in the system.

Reply:
@frkv : For which boards was MBEDTLS_BUILTIN causing trouble? If it was a non-secure board then that should be fixed by the patches I have in PR.

Reply:
It will cause troubles on all nRF-boards if nrf_security is used. 

If you have another fix for this, then I'm happy to accept that!

Reply:
NRF_SECURITY is doing
	select DISABLE_MBEDTLS_BUILTIN if MBEDTLS

so I don't think this patch is necessary, I suggest we remove this patch and re-test.

Reply:
will look into this

Reply:
this sample is not building with any nrf board I can find.

So I am unable to test the patch.

on NCS main it overflows, in this PR it has a Kconfig issue.

Will debug.

Reply:
> @frkv I've already fixed #722 for existing patches, but it's `[nrf noup] subsystem: blah blah blah`, not `[nrf_noup]: subsystem: blah blah blah`; please fix

Will fix

Reply:
Fixed and cleaned up the commit substantially. Now it only checks for nrf_security being enabled and adds a minor fix in the config-file to prevent mbedtls_platform_zeroize from being duplicated (present in nrf_cc3xx_platform)